docs/src/api/class-worker.md

```diff
@@ -58,6 +58,16 @@ foreach(var pageWorker in page.Workers)
 
 Emitted when this dedicated [WebWorker](https://developer.mozilla.org/en-US/docs/Web/API/Web_Workers_API) is terminated.
 
+## event: Worker.console
```

Comment:
Should we dispatch it on the BrowserContext/Page instead and have a property indicating that it is from a service worker?

Reply:
Lets defer that to API review.